.rhcicd/pr_check.sh

```diff
@@ -46,7 +46,7 @@ export APP_NAME="notifications"
 export COMPONENT_NAME="notifications-backend"
```

Comment:
Could you please add a few lines above to build the pr-check image and push it to Quay?
```
# Build the notifications-connector-email image and push it to Quay
export IMAGE="quay.io/cloudservices/notifications-connector-email"
export DOCKERFILE=docker/Dockerfile.notifications-connector-email.jvm
source $CICD_ROOT/build.sh
```

Reply:
Added!